scripts/build.py

```diff
@@ -1097,6 +1097,9 @@ def generate_build_directory():
 
 
 def build():
+    """Creates a third party directory where all the dependencies of JS and
```

Comment:
Doesn't this also do other things based on the supplied options? E.g. it generates a build directory too if we are doing things in prod mode, etc.

So perhaps let's do something like this (based off your current description):

```
"""The main method of this script.

Creates a third-party directory where all the JS and CSS dependencies are built and stored. Depending on the options passed to the script, might also minify third-party libraries and/or generate a build directory.
"""
```

Reply:
ok sir.